packages/di/src/common/decorators/intercept.spec.ts

```
@@ -49,7 +70,31 @@
       expect(getInterceptorOptions(ServiceTest, "exec")).toEqual("options data");
 
       // THEN
-      expect(result).toEqual("Original data - param data - options data - intercepted");
+      expect(result).toEqual("Original data - param data - options data - intercepted 1");
+    });
+    it("should chained interceptor", async () => {
+      // GIVEN
+      const serviceTest = await DITest.invoke<ServiceTest>(ServiceTest)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)